iceoryx2-pal/posix/src/macos/fcntl.rs

```diff
@@ -44,7 +46,19 @@ pub unsafe fn fcntl2(fd: int, cmd: int) -> int {
 }
 
 pub unsafe fn fchmod(fd: int, mode: mode_t) -> int {
-    crate::internal::fchmod(fd, mode)
+    // TODO iox2-156, shared memory permission cannot be adjusted with fchmod, therefore setting
```

Comment:
This is safe since the dynamic storage uses always two mechanisms to ensure that the underlying construct is initialized.
1. permissions
2. the first 8 bytes where the version number is stored

Reply:
I would love to, but the logger is two layers above this. To do this, we need the logger below the posix platform abstraction, like in iceoryx1.

The only thing I can do is to print it directly to the console, but then the user is spammed with a lot of useless warning messages.

Reply:
Are you able to redirect the output of the log crate to the console logger? If yes, then this could become our interface. Not in this PR of course.